addons/point_of_sale/models/pos_order.py

```diff
@@ -134,13 +134,7 @@ def _link_combo_items(self, combo_child_uuids_by_parent_uuid):
     def _process_saved_order(self, draft):
         self.ensure_one()
         if not draft:
-            try:
-                self.action_pos_order_paid()
-            except psycopg2.DatabaseError:
-                # do not hide transactional errors, the order(s) won't be saved!
-                raise
-            except Exception as e:
-                _logger.error('Could not fully process the POS Order: %s', tools.exception_to_unicode(e))
+            self.action_pos_order_paid()
```

Comment:
We discuss about that with JCB, can you do it in master ?:)

Reply:
@davidmonnom I can apply the fix in the master branch, but we also need to find a proper solution for this issue.

Even if we decide to ignore the error in `action_pos_order_paid`, we should not allow the creation of an order that isn’t fully paid.

This issue occurs specifically when a payment is removed from another device, and the order is not updated accordingly. Let me know how you’d like to proceed.

Reply:
@caburj